wazuh/wazuh-kibana-app
# Redesign the SCA table from agent's dashboard

With the redesign of this section, the user accesses the information in a much more direct way.

This PR closes this [issue](https://github.com/wazuh/wazuh-kibana-app/issues/4238).

## For testing

### Check the new SCA Lastest scans table in the Agent Dashboard
1. Select Agent and go to `Agent Dashboard`
2. Check table style in the `SCA: Lastest scans` card.
3. Click on the table row and check if redirect to the `SCA > Inventory` to the selected policy.

### Check the SCA > Inventory behavior
1. Select Agent and go to `Agent Dashboard`
2. Click on the SCA title in the Header navigation ( `SCA > Inventory`)
3. Check the `Policies table` is rendered on the `Inventory Tab`.
4. Click in the `Policy Table Row` and check if is redirected to the` Policy Checks` view
